garden-backend-service/.flake8

```diff
@@ -1,3 +1,4 @@
 [flake8]
 max-line-length = 150
-extend-ignore = E203
+extend-ignore = E203, PT004
+exclude = migrations,__pycache__,schemas
```

Comment:
why ignore schemas? (assuming this matches `src/api/schemas`)

Reply:
Oh, whoops! I added this to my local config while working on this to suppress some flake8 complaints while I worked on the tests. I meant to take it out before pushing.